docs/troubleshooting.md

```diff
@@ -50,3 +55,43 @@ If no volumes are in mounted state and `lsscsi` lists any 3PAR data volumes then
 for i in `lsscsi | grep 3PARdata | awk '{print $6}'| grep -v "-"| cut -d"/" -f3`; do echo $i; echo 1 > /sys/block/$i/device/delete; done
 rescan-scsi-bus.sh -r -f -m
 ```
+
+### Collecting necessary Logs
+
+if any issue found please do collect following logs from your Docker host
+
+```
+v3.1 onwards 
+/etc/hpedockerplugin/3pardcv.log
+```
+#### Managed Plugin
+for any older version below v3.1
+
+```
+/var/log/messages
+```
+#### Containerized Plugin 
+
+```
+$docker logs -f <container id of Plugin> 
+container id of plugin can got by $docker ps -a | grep hpe 
+```
+
+ ## Capturing Logs in Kubernetes/OpenShift environments
+ 
+ Collect above Containerized Plugin logs along with than following logs are required.
```

Comment:
Collect above Containerized Plugin logs along with the following logs.

Reply:
done updated typo